app/controllers/categories_controller.rb

```diff
@@ -1,18 +1,21 @@
 class CategoriesController < ApplicationController
 
-  def new
-
-  end
-
   def create
-    Category.create(category_params)
+    Category.create(name: params[:categories][:name],
+                    image: choose_icon)
     redirect_to items_path
   end
 
-
   private
 
-  def category_params
-    params.require(:categories).permit(:name)
+  def choose_icon
+    icon = icons.shuffle.pop
+    params[:image] = icon
+  end
+
+  def icons
+    ['fa fa-crosshairs', "fa fa-anchor", "fa fa-bolt", "fa fa-connectdevelop",
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.